src/System.Private.ServiceModel/tools/setupfiles/BridgeCertificateBootstrapper.cmd

```diff
@@ -0,0 +1,25 @@
+echo off
+setlocal
+
```

Comment:
Nit: possibly some REM's up front that describe intended usage of this CMD. I usually forget them...